File: LinguaRise/LinguaRise.Models/DTOs/Course/CourseDTO.cs
namespace LinguaRise.Models.DTOs;

public class CourseDTO
{
    public int Id { get; set; }
    public Guid? UserId { get; set; }
    public string? UserName { get; set; }
    public string? UserEmail { get; set; }
    public int? LanguageId { get; set; }
    public string? LanguageCode { get; set; }
    public string? LanguageName { get; set; }
    public ICollection<LessonDTO> Lessons { get; set; } = new List<LessonDTO>();
}

File: LinguaRise/linguarise.app/src/data/CourseDTO.js
export class CourseDTO {
  constructor({ userId = null, languageId = null } = {}) {
    this.id = 0;
    this.userId = userId;
    this.userName = null;
    this.userEmail = null;
    this.languageId = languageId;
    this.languageCode = null;
    this.languageName = null;
    this.lessons = [];
  }
}